taskcluster/ci/docker-image/kind.yml

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.docker_image:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+jobs:
+    k8s-image:
+        definition: k8s-image
+    python38:
+        definition: python
+        args:
+            PYTHON_VERSION: "3.8"
+    python37:
+        definition: python
+        args:
+            PYTHON_VERSION: "3.7"
+    pushapkscript-py38:
```

Comment:
~Hm, question: how come there's only three scriptworkers here?~
Later edit: don't mind me, just realized by looking at the docker images later on that they all three have externals deps that are different than the base image that we build for the others ones.